docs/send-data/opentelemetry-collector/troubleshooting-and-faqs.md

```diff
@@ -0,0 +1,326 @@
+---
+id: troubleshooting-and-faqs
+title: Troubleshooting and Faqs
+sidebar_label: Troubleshooting and Faqs
+description: Troubleshooting and Faqs
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This document contains common troubleshooting scenarios and frequently asked questions about Sumo Logic Open Telemetry Collector from our customers and field teams (SE, TAM, Support Engineers).
+
+<details><summary>Solutions to OpenTelemetry Connection Issues</summary>
+
+## Collector fails to connect to Sumo Logic
+
+### Question
```

Comment:
Please convert the questions/answers themselves into headers (rather than "Question"/"Answer" as headers). You can view the formatting here: https://help.sumologic.com/docs/send-data/collector-faq.

Reply:
They are headers only. I checked the collector FAQs page before starting. If you mean something else, can you please specify?

Reply:
The questions themselves must be the headers -- like this:

```
## Collector fails to connect to Sumo Logic

### How do I provide a PROXY setting to Sumo Logic OpenTelemetry Collector so that it can connect to Sumo Logic Backend?

### On Windows, I am running the Collector install command on CMD. Why does it fail?
```

Reply:
But if you see the Collector FAQs doc, it's like:
### Question 
When attempting to upgrade a Windows Collector from the UI, the upgrade fails and the following error is reported in the tooltip of the failed Collector:

### Answer 
This is a known issue regarding upgrading a Windows collector from versions 19.60-x to the latest released version. The cause is a missing wrapper.dll file, which is required during a pre-check test of the Collector during upgrade.